### ISSUE TITLE
Is the Default trait implemention missing of strct DEVMODEA?

### ISSUE THREAD

**Issue by luojunyuan** [microsoft/windows-rs#1145]
I manage to use [EnumDisplaySettingsA](https://microsoft.github.io/windows-docs-rs/doc/bindings/Windows/Win32/Graphics/Gdi/fn.EnumDisplaySettingsA.html) but find the parameter `lpdevmode` can not be initialized like `DEVMODEA::default()`, I'm new to rust. Should I initialize it manually like `let lpdevmode = DEVMODEA { bla };`

https://microsoft.github.io/windows-docs-rs/doc/bindings/Windows/Win32/UI/DisplayDevices/struct.DEVMODEA.html#trait-implementations

~sry for poor english~

**Comment by kennykerr** [comment]
`Default` isn't currently provided for unions or structs containing unions such as `DEVMODEA`. I hope to address that soon. 

For now, you can use `std::mem::zeroed` to achieve the same thing:

```rust
fn main() {
    unsafe {
        let mut mode = std::mem::zeroed();
        EnumDisplaySettingsA(None, ENUM_CURRENT_SETTINGS, &mut mode);
        println!("dmBitsPerPel {}", mode.dmBitsPerPel);
        println!("dmPelsWidth {}", mode.dmPelsWidth);
        println!("dmPelsHeight {}", mode.dmPelsHeight);
        println!("dmDisplayFrequency {}", mode.dmDisplayFrequency);
    }
}
```

**Comment by kennykerr** [comment]
The `Default` trait should now be available as of #1146.
